userguide/plugins.rst

```diff
@@ -101,16 +101,19 @@ with :guilabel:`NAT`. It is recommended to keep these plugins set to
 :guilabel:`DHCP` unless manually configuring an IP address is preferred.
 
 If both :guilabel:`NAT` and :guilabel:`DHCP` are unset, an IPv4 or IPv6 address can
-be manually entered. 
+be manually entered.
 
 Click :guilabel:`ADVANCED PLUGIN INSTALLATION` to show all options for the
 plugin jail. The options are described in :ref:`Advanced Jail Creation`.
 
 To start the installation, click :guilabel:`SAVE`.
 
-Depending upon the size of the application, the installation may take a few minutes
-to download and install. A confirmation message displays once the plugin installation
-completes, along with any post-installation notes. 
+Depending upon the size of the application, the plugin can take several
+minutes to download and install. A confirmation message displays when
+the plugin installation completes, along with any post-installation
+notes. Installed plugins are immediately activated and set to start when
```

Comment:
What does "immediately activated" mean?  Started after install?  If so, this should be a separate sentence.  The next two sentences can be about how they default to autostart, but can be changed.